tools/publish_release.py

```diff
@@ -61,7 +61,7 @@ def release_channel():
   return channel
 
 def publish_release(github, release_id):
-  data = dict(draft=False)
+  data = dict(draft=False, prerelease=True)
```

Comment:
Oh funny, not that I'm aware of @luixxiul . I based off of master, so I'm not sure why that deviated. 

@bsclifton should we default to prerelease like that?